### DISCUSSION TITLE
Custom Scan for Antivirus

### DISCUSSION THREAD

**Discussion by Greek-Fire** [ComplianceAsCode/content#11741] [Q&A]
Good morning from Knoxville, TN.

Does anybody have a good example of altering the scan of a STIG Rule? McAfee is EOF for Linux, and I would like to either create a custom rule with a custom scans/OVAL check, or alter the rule that already exists for McAfee.

CCE-83879-7

**Comment by Mab879** [comment]
Thanks for your question.

If you have the package name of the most current solution it's fairly easy to create a rule for that. 

See  the [package_sudo_installed](https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/software/sudo/package_sudo_installed/rule.yml) rule as an example.

If you looking for a more complicated rule, like a service running (that is also easy see [service_sshd_enabled](https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/services/ssh/service_sshd_enabled/rule.yml)) or configure options we can help with that as well.

If you need any help with that let us know.

**Reply by Greek-Fire** [reply]
Thank you for responding so quickly. I am going to try somethings and report back. when I figure it out, since you all are doing a fantastic job of communicating to your fan base,  would there be a place for me to describe the steps to do this in more detail?

**Reply by Mab879** [reply]
As reply to this discussion is a fine place. 

You can also talk to us on [Gitter](https://gitter.im/Compliance-As-Code-The/content?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).
